composer.json

```diff
@@ -21,5 +21,10 @@
             "" : "src/"
         }
     },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "2.0.x-dev"
+        }
+    },
```

Comment:
yes, so that you can require this bundle using 1.* for every 1.x tags and use 2.* for this release